### PR TITLE
Time measurement

### PR DESCRIPTION
**Pull request by qualmluc**
I created a pull request with
- the dp algorithm
- time measurement methods
- unit resolution applied on dpll

### CODE REVIEW

**Comment by qualmluc** [comment]
feel free to test the code on a free minute or so and request changes, if something is odd for you in the implementations

File: src/alg/bruteforce.py
Comment (melanie-taprogge) [inline src/alg/bruteforce.py:1]: looks good! I just notice that there are still a few commented "print"-commands that I suppose you used to check stuff while writing the code! (lines 17, 26, 46).

File: src/alg/dp.py
Comment (melanie-taprogge) [inline src/alg/dp.py:1]: looks like a nice implementation of the algorithm described in the handbook to me!

File: src/timing/measure.py
Comment (melanie-taprogge) [inline src/timing/measure.py:1]: looks good to me!

**Review by melanie-taprogge — APPROVED** [review]
All looks very well to me! :)
I just found a few commented "print" commands that were not deleted (see comments).

**Review by JWittkat — APPROVED** [review]

**Pull request merged**